changelog/24263.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+core: Provide management endpoints for UI Custom Messages
```

Comment:
For `feature` changelogs, there's a different format (in particular, `**Feature Name**: Description`). However, for a feature like this, it might make sense to add a single feature changelog once the whole thing is in. Up to you how you want to handle it!

Reply:
In case it helps: I think what I'd do is add a `release-note:feature` changelog with the docs PR, but not use a changelog for the others, as the feature is all of the changes together.

Reply:
Ok, I'll remove this file and adjusted the PR labels

Reply:
It's gone now. Thanks.